.gitattributes

```diff
@@ -1,3 +1,4 @@
 docker/lfs/* filter=lfs diff=lfs merge=lfs -text
 installer/alpine/assets-*/* filter=lfs diff=lfs merge=lfs -text
 installer/alpine/eclypsiumdriver-alpine-2.0.0.tgz filter=lfs diff=lfs merge=lfs -text
```

Comment:
here too, maybe change to `installer/alpine/eclypsiumdriver-alpine* ...`?

Reply:
Yeah, good call. Fixed.